Podfile

```diff
@@ -9,7 +9,8 @@ pod 'SCLAlertView'
 pod 'ObjectMapper'
 pod 'SwiftyJSON'
 pod 'DKImagePickerController'
-pod "PKHUD"
+pod 'PKHUD'
+pod 'KTCenterFlowLayout'
 
```

Comment:
Good PR overall. Left a note to respond to a comment you had that addressed me. Look over things and merge when you feel like it.